src/compressed_tensors/compressors/quantized_compressors/pack_quantized.py

```diff
@@ -136,6 +136,18 @@ def pack_to_int32(value: torch.Tensor, num_bits: int) -> torch.Tensor:
     """
     Packs a tensor of quantized weights stored in int8 into int32s with padding
 
+    Pseudocode:
+     1. Shift wrt num_bits to convert to unsigned. num_bits=8
+        [1,2] -> [129, 130]
+     2. Pad to fill in 32 bits
+        [129, 130] -> [129, 130, 0, 0]
+     3. convert to binary align in order
+        [129, 130, 0, 0] -> 00000000 00000000 10000010 10000001
+     4. convert aligned binary to number
+        00000000000000001000001010000001 -> 33409
+     5. covert back to uint32
+        33409 -> 33409
+
     :param value: tensor to pack
     :param num_bits: number of bits used to store underlying data
```

Comment:
Could we update the num_bits param to say it should be a positive number? Since the change would now fail for num_bits=0; (which should never happen ideally but it's good to be explicit)